rpcs3/CMakeLists.txt

```diff
@@ -41,6 +41,15 @@ file(GLOB RPCS3_SRC "*.cpp")
 
 if(WIN32)
 	add_executable(rpcs3 WIN32 ${RPCS3_SRC})
+elseif(APPLE)
+	set(MACOSX_BUNDLE_BUNDLE_NAME rpcs3)
+	set(MACOSX_BUNDLE_GUI_IDENTIFIER "net.rpcs3.rpcs3")
+	set(MACOSX_BUNDLE_INFO_STRING "Open-source Sony PlayStation 3 emulator")
+	set(MACOSX_BUNDLE_ICON_FILE "rpcs3.icns")
+	set(MACOSX_BUNDLE_LONG_VERSION_STRING "1.0")
```

Comment:
Unless this is a bundle specification version, I think this should be RPCS3's version, which is at 0.0.5 right now.

Reply:
The reason I chose 1.0 is that I do not seem to be able to tie this version to rpcs3 version within cmake. I will recheck, but if it is the case, I do not believe it is reasonable to hardcode this information here. We will simply forget to update it.

Reply:
Perhaps it would be best to set a global rpcs3 version variable at the start of CMakeLists.txt, in case we want to use it for a windows installer down the line, and also should be easier to remember to update it when the version gets bumped.

One alternative would be parsing rpcs3/rpcs3_version.cpp but I'm thinking that would be too much of a pain and too easy to break.

Reply:
My personal opinion initially was that we should define a macro within the CMakeLists.txt and pass it to the code (like `-DRPCS3_VERSION=0.0.5`), but this idea is not perfect, as Windows build system does not seem to be cmake based.

I believe the only solution in this direction is to have a separate VERSION file with 0.0.5 contents. This file will be used in generation of rpcs3_version.cpp/a file it includes, similarly to git version file. Such a method will require interaction with Windows buildsystem, particularly Visual Studio, which I do not have nearby, so I added a TODO.